### PR TITLE
[v2] Scope terra-application

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

convert terra application to a scoped package `@cerner/terra-application`

Closes #

### Deployment Link

https://terra-applic-.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by emilyrohrbough — APPROVED** [review]

File: src/application-navigation/terra-application-navigation/utility-menu/_UtilityMenu.jsx [hunk 1]
```diff
@@ -58,8 +58,8 @@ const defaultProps = {
   utilityItems: [],
 };
 
-const utilityMenuSettingsKey = 'terra-application-navigation.utility-menu.settings';
-const utilityMenuHelpKey = 'terra-application-navigation.utility-menu.help';
+const utilityMenuSettingsKey = '@cerner/terra-application-navigation.utility-menu.settings';
+const utilityMenuHelpKey = '@cerner/terra-application-navigation.utility-menu.help';
```
Comment (StephenEsser) [inline src/application-navigation/terra-application-navigation/utility-menu/_UtilityMenu.jsx]: These non-import keys probably don't need the scope

Comment (mjhenkes) [reply src/application-navigation/terra-application-navigation/utility-menu/_UtilityMenu.jsx]: lol, thats what you get when you find and replace

File: tests/jest/application-navigation/terra-application-navigation/utility-menu/__snapshots__/UtilityMenu.test.jsx.snap [hunk 2]
```diff
@@ -217,7 +217,7 @@ exports[`UtilityMenu should render with function callbacks 1`] = `
             viewBox="0 0 48 48"
             xmlns="http://www.w3.org/2000/svg"
           />,
-          "key": "terra-application-navigation.utility-menu.settings",
+          "key": "@cerner/terra-application-navigation.utility-menu.settings",
```
Comment (mjhenkes) [inline tests/jest/application-navigation/terra-application-navigation/utility-menu/__snapshots__/UtilityMenu.test.jsx.snap]:
```suggestion
          "key": "terra-application-navigation.utility-menu.settings",
```

File: tests/jest/application-navigation/terra-application-navigation/utility-menu/__snapshots__/UtilityMenu.test.jsx.snap [hunk 3]
```diff
@@ -228,7 +228,7 @@ exports[`UtilityMenu should render with function callbacks 1`] = `
             viewBox="0 0 48 48"
             xmlns="http://www.w3.org/2000/svg"
           />,
-          "key": "terra-application-navigation.utility-menu.help",
+          "key": "@cerner/terra-application-navigation.utility-menu.help",
```
Comment (mjhenkes) [inline tests/jest/application-navigation/terra-application-navigation/utility-menu/__snapshots__/UtilityMenu.test.jsx.snap]:
```suggestion
          "key": "terra-application-navigation.utility-menu.help",
```

File: tests/jest/application-navigation/terra-application-navigation/utility-menu/__snapshots__/UtilityMenu.test.jsx.snap [hunk 4]
```diff
@@ -496,7 +496,7 @@ exports[`UtilityMenu should render with function callbacks 1`] = `
                         xmlns="http://www.w3.org/2000/svg"
                       />
                     }
-                    key="terra-application-navigation.utility-menu.settings"
+                    key="@cerner/terra-application-navigation.utility-menu.settings"
```
Comment (mjhenkes) [inline tests/jest/application-navigation/terra-application-navigation/utility-menu/__snapshots__/UtilityMenu.test.jsx.snap]:
```suggestion
                    key="terra-application-navigation.utility-menu.settings"
```

File: tests/jest/application-navigation/terra-application-navigation/utility-menu/__snapshots__/UtilityMenu.test.jsx.snap [hunk 5]
```diff
@@ -579,7 +579,7 @@ exports[`UtilityMenu should render with function callbacks 1`] = `
                         xmlns="http://www.w3.org/2000/svg"
                       />
                     }
-                    key="terra-application-navigation.utility-menu.help"
+                    key="@cerner/terra-application-navigation.utility-menu.help"
```
Comment (mjhenkes) [inline tests/jest/application-navigation/terra-application-navigation/utility-menu/__snapshots__/UtilityMenu.test.jsx.snap]:
```suggestion
                    key="terra-application-navigation.utility-menu.help"
```

File: src/application-navigation/terra-application-navigation/utility-menu/_UtilityMenu.jsx (hunk 1 above)
Comment (mjhenkes) [inline src/application-navigation/terra-application-navigation/utility-menu/_UtilityMenu.jsx]:
```suggestion
const utilityMenuSettingsKey = 'terra-application-navigation.utility-menu.settings';
const utilityMenuHelpKey = 'terra-application-navigation.utility-menu.help';
```

**Comment by mjhenkes** [comment]
Theres links in here that are going to need to be updated too.

**Review by tbiethman — APPROVED** [review]

**Pull request merged**